src/attr/_make.py

```diff
@@ -279,27 +279,25 @@ def _transform_attrs(cls, these, auto_attribs):
     # redefinitions deeper in the hierarchy.
     super_attrs = []
     taken_attr_names = {a.name: a for a in non_super_attrs}
-    for super_cls in cls.__mro__[1:-1]:
+    for super_cls in cls.__mro__[1:]:
```

Comment:
comment about this change: what if the last one isn't object, i.e. why to impose assumptions without validation?  If `object`, it would be happily skipped since wouldn't have any `__attrs_attrs__`